### DISCUSSION TITLE
SuperGluu Update iOS

### DISCUSSION THREAD

**Discussion by quer0016** [JanssenProject/jans#7845] [Q&A]
Hello team, 
We have recently received an update of the super gluu app for iOS, the problem we have detected is that the users who have updated the APP have problems logging into their account, the notification arrives but it does not validate, it is solved by removing this user device and putting it back as 2fa supergluu and it works again. Is it normal for this to happen with app updates?

**Comment by nynymike** [comment]
No... the user should not have to re-enroll. We're pushing a fix to the app.

**Reply by quer0016** [reply]
thx
